Tasks/Zhukovets/Task1/task1.py

```diff
@@ -0,0 +1,7 @@
+initial_amount = 20000
+time = 5
+percent = 15
+number_of_months = 5 * 12
```

Comment:
these values should be of the decimal data type, since they are money-related, -1 point for the unwise choice